AlloyReact/Static/react/editable.js

```diff
@@ -0,0 +1,60 @@
+import hoistStatics from 'hoist-non-react-statics';
+import React, { Component } from 'react';
+
+const defaultValueUpdator = (value, propName, props, propertyDetails = null) => {
+    return {
+        [propName]: value
+    }
+}
+
+const editable = (WrappedComponent, onValueChange = defaultValueUpdator) => {
```

Comment:
I like the idea! Should we update the other examples (`ThreeKeyFacts.jsx` and `PageListBlock.jsx`) to use this as well?

Reply:
Yes we definitely should